dotenv-linter/tests/flags/check_updates.rs

```diff
@@ -26,3 +27,17 @@ fn print_new_version_if_nothing_to_check() {
     let args: &[&str; 0] = &[];
     test_dir.test_command_success_with_args(args, expected_output);
 }
+
+#[test]
+fn do_not_print_new_version() {
+    env::set_var("DOTENV_LINTER_NOT_CHECK_UPDATES", "true");
```

Comment:
It also would be nice to pass this env to `cmd` ([doc](https://doc.rust-lang.org/std/process/struct.Command.html#method.env)), otherwise there is a chance of parallel run with other tests with enabled variable. So it could silently affect other tests.

Take a look at `TestDir`, we need to extend it to allow passing `ENV` variables for commands.

Reply:
I added a commit for this here: https://github.com/dotenv-linter/dotenv-linter/pull/625/commits/6ff87026df99705446705fc338333d81e27a81fe

I am not too sure if I am doing it the right way though. It feels like I am making too many changes, please let me know if there's an obvious way I am missing! 🙇

Reply:
Here the possible options are: https://github.com/dotenv-linter/dotenv-linter/pull/625#discussion_r1070291629